device_services/meta/parameters.md

```diff
@@ -183,3 +183,5 @@ However, there are some predefined parameters which have a standardized meaning
 | `device_lock`             | Device lock             | `thermostat`,`power_regulator` | `string`   | `select`    | `unlocked`, `locked`, `locked_with_pin`                | Configure lock setting of the thermostat device.                                                                                                                                                                                            |
 | `used_sensors`            | Used sensors            | `thermostat`                   | `string`   | `select`    | `room_sensor`, `floor_sensor`, `floor_and_room_sensor` | Select which sensors should be used by the thermostat to calculate heating hysteresis.                                                                                                                                                      |
 | `floor_sensor_resistance` | Floor sensor resistance | `thermostat`                   | `int`      | `select`    |                                                        | Set the floor sensor resistance in `kΩ`, if it is installed.                                                                                                                                                                                |
+| `low_current_time`        | Low current time        | `chargepoint`                  | `int`      | `input`     |                                                        | Set the time after which the low current detection feature will stop an ongoing transaction in minutes.                                                                                                                                                                                |
+| `low_current_threshold`   | Low current threshold   | `chargepoint`                  | `int`      | `input`     |                                                        | Stops the transaction if for a prolonged time the EV is charging with a low current.                                                                                                                                                                                |
```

Comment:
Maybe: ...with a current lower than configured number of Ampers... or similar. There is no information we set the threshold to ampers. Please also check which one when set to 0 will disable the feature altogether.

Reply:
We some of settings are controled by chargepoint service and rest of them (specific ones) by parameters service ?

Reply:
fixed